app/code/Magento/InventoryApi/Test/_files/source_items.php

```diff
@@ -57,6 +57,12 @@
         SourceItemInterface::QUANTITY => 5,
         SourceItemInterface::STATUS => SourceItemInterface::STATUS_IN_STOCK,
     ],
+    [
```

Comment:
Need to update comment above source items declaration
```
/**
 * SKU-1 - EU-source-1(id:10) - 5.5qty
 * SKU-1 - EU-source-2(id:20) - 3qty
 * SKU-1 - EU-source-3(id:30) - 10qty (out of stock)
 * SKU-1 - EU-source-4(id:40) - 10qty (disabled source)
 *
 * SKU-2 - US-source-1(id:30) - 5qty
 */
```

Reply:
done